templates/blog/post_detail.html

```diff
@@ -0,0 +1,88 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <div class="container top-container">
+    <div class="row">
+      {% block blog_logo %}
+        {% include "includes/blog_logo.html" %}
+      {% endblock %}
+
+      <div class="col-8 col-offset-2">
+        <h1>{{ post.title }}</h1>
+        {% if post.owner.first_name or post.owner.last_name %}
+          <h2>{{ post.owner.first_name }} {{ post.owner.last_name }}</h2>
+        {% else %}
+          <h6>{{ post.owner }}</h6>
+        {% endif %}
+          <p style="font-weight: lighter">{{ post.created_time|date:"d M, Y H:m" }}</p>
+      </div>
+      <div class="col-2">
+
+        {% block log_in_out %}
+          {% include "includes/log_in_out.html" %}
+        {% endblock %}
+
+      </div>
+    </div>
+  </div>
+
+  <div class="container">
+    <div class="row">
+        <div class="card" style="margin-bottom: 5px; padding: 10px">
+          {{ post.content }}
+        </div>
+    </div>
+  </div>
+
+  <div class="container col-10" style="margin: auto;">
+    <div class="card">
+      <div class="card-header comment">
+        {% if post.comments.all|length %}
+        {{ post.comments.all|length }} comment{{ post.comments.all|pluralize }}:
+        {% else %}
+          There are no comments yet
+        {% endif %}
+
+      </div>
+      <div class="card" style="margin: 10px 0">
+        {% if post.comments %}

```

Comment:
The condition `{% if post.comments %}` is unnecessary because the `for` loop will naturally handle an empty queryset. You can safely remove this condition to simplify the code.